examples/pid_lower_than_1000.yaml

```diff
@@ -0,0 +1,13 @@
+name: pid_lower_than_1000
+description: This policy traces read and write syscalls for pids lower than 1000
+scope: 
+  - global
+defaultAction: log
+rules:
+  - event: read
+    filter: 
+      - pid<1000
+  - event: write
+    filter:
+      - pid<1000
```

Comment:
just to justify the use of event context filter and not a scope, better have different pid filters for the read and write events